src/less/details/ds4/details.less

```diff
@@ -0,0 +1,38 @@
+@import "../../less/ds4/variables.less";
+
+details.details:not([open]) {
+    .details__summary--showless {
+        display: none;
+    }
+}
+
+details.details[open],
+details.details[data-makeup-open] {
```

Comment:
@ianmcburnie I'll need to check if this is necessary

Reply:
Yeah, `data-makeup-open` was only need for my basic polyfill. I think the polyfill you are using always uses the `open` property.